units/pod_allocator_test.go

```diff
@@ -305,7 +305,7 @@ func TestPodAllocatorJob(t *testing.T) {
 			env.EvergreenSettings.Providers.AWS.Pod.ECS.AllowedImages = []string{
 				"rhel",
 			}
-			require.NoError(t, env.EvergreenSettings.Providers.Set())
+			require.NoError(t, env.EvergreenSettings.Providers.Set(tctx))
```

Comment:
What is tctx?

Reply:
The way I understood it it looked like we do some initial setup with [one context](https://github.com/evergreen-ci/evergreen/blob/3ede2d9ecf509f4c3e95a7e3479a6df1bd2dc55c/units/pod_allocator_test.go#L26-L27) and then create `tctx` as [a child with a timeout](https://github.com/evergreen-ci/evergreen/blob/3ede2d9ecf509f4c3e95a7e3479a6df1bd2dc55c/units/pod_allocator_test.go#L257-L258) for execution of each of the individual tests.